xrpl/asyncio/ledger/main.py

```diff
@@ -74,18 +86,66 @@ async def get_fee(
     if not response.is_successful():
         raise XRPLRequestFailureException(response.result)
 
-    result = response.result["drops"]
-    if fee_type == "open":
-        fee = cast(str, result["open_ledger_fee"])
-    elif fee_type == "minimum":
-        fee = cast(str, result["minimum_fee"])
-    else:
-        raise XRPLException(
-            f'`fee_type` param must be "open" or "minimum". {fee_type} is not a '
-            "valid option."
+    result = response.result
+    drops = result["drops"]
+    if fee_type:
+        if fee_type == "open":
+            fee = cast(str, drops["open_ledger_fee"])
+        elif fee_type == "minimum":
+            fee = cast(str, drops["minimum_fee"])
+        else:
+            raise XRPLException(
+                f'`fee_type` param must be "open" or "minimum". {fee_type} is not a '
+                "valid option."
+            )
+    else:  # use https://gist.github.com/WietseWind/3e9f9339f37a5881978a9661f49b0e52
+        current_queue_size = int(result["current_queue_size"])
+        max_queue_size = int(result["max_queue_size"])
+        queue_pct = current_queue_size / max_queue_size
+        minimum_fee = int(drops["minimum_fee"])
+        median_fee = int(drops["median_fee"])
+        open_ledger_fee = int(drops["open_ledger_fee"])
```

Comment:
If we extract this "else" section into it's own function with these as parameters, it'll let you write unit tests for this logic :) 

Some basic things that would be good to test:
* When given `queue_pct = .5` - Is that fee more than when given `queue_pct = 0`
* Similarly when `queue_pct = 1` is that fee more than `queue_pct = .9`

Reply:
Good idea 👍 I don't think it would make sense to add it to the async folder. I would add it to utils. What do you think? :)

Reply:
It should still be in this folder, because it's only a helper function used here, not a standalone method.